### ISSUE TITLE
Impossible to obtain Let's encrypt certs on NS6

### ISSUE THREAD

**Issue by filippocarletti** [NethServer/dev#5389]
I can no longer obtain or renew LE certificates after 15 November 2017 on NethServer 6.9.

**Steps to reproduce**

- `/usr/libexec/nethserver/letsencrypt-certs -t`

**Expected behavior**

No errors and a certificate. :-)

**Actual behavior**

```
  + ERROR: An error occurred while sending post-request to https://acme-staging.api.letsencrypt.org/acme/new-reg (Status 400)

Details:
{
  "type": "urn:acme:error:malformed",
  "detail": "Provided agreement URL [https://letsencrypt.org/documents/LE-SA-v1.1.1-August-1-2016.pdf] does not match current agreement URL [https://letsencrypt.org/documents/LE-SA-v1.2-November-15-2017.pdf]",
  "status": 400
}
```

**Comment by gsanchietti** [comment]
In nethserver-testing (6.9):
nethserver-letsencrypt-1.0.4-1.4.g6503eb7.ns6.noarch.rpm
dehydrated-0.4.0-1.el6.noarch.rpm

**Comment by gsanchietti** [comment]
Released (6.9):
- dehydrated-0.4.0-1.el6.noarch.rpm
- nethserver-letsencrypt-1.0.5-1.ns6.noarch.rpm

**Comment by DavidePrincipi** [comment]
Added reference in the ns6 bugtracker:
http://dev.nethserver.org/issues/3442
